ckanext/aberdeen/actions.py

```diff
@@ -1,13 +1,20 @@
 import logging
-
+from datetime import datetime, timedelta
+import threading
 import sqlalchemy as sqla
+from pylons import config
+
+import json
+import requests
```

Comment:
Redundant

Reply:
@zelima Just pushed the changes. I need to keep `from pylons import config` to check the config file though